### PR TITLE
[GuestLoader]: Add autoboot for guestloader

### PR DESCRIPTION
**Pull request by in1004kyu**
1. Add CLI (command.*)
2. Add autoboot (main.c)
3. Fix guestloader's memory map(Add sp for system/user mode)

CLI is worked on RTSM. Arndale is not supproted.

### CODE REVIEW

**Comment by kesl** [comment]
CI : Test PASSed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/127/

File: common/guest/core/gic.c [hunk 1]
```diff
@@ -198,8 +197,6 @@ hvmm_status_t gic_init(void)
     if (result == HVMM_STATUS_SUCCESS)
         _gic.initialized = GIC_SIGNATURE_INITIALIZED;
 
-    /* enable virtual timer */
-    vtimer_mask(0);
```
Comment (in1004kyu) [inline common/guest/core/gic.c:202]: This line move to c_start.c. See above

File: common/guest/command/command.c [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    load_guest(GUEST_TYPE);
+}
+
+void command_banner(int argc, char **argv)
+{
+    uart_print("command_show : show banner\n");
+    uart_print(BANNER_STRING);
+}
+
+/* @todo Not implemeted yet */
```
Comment (in1004kyu) [inline common/guest/command/command.c]: I will fix this typographical error next time.

**Comment by joohyun0115** [comment]
Look so good!

File: common/guest/command/command.h [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#ifndef __COMMAND_H__
+#define __COMMAND_H__
+/**
+ * @brief Executes command.
+ *
+ * help (h)        - List commands and their usage
+ * boot (h)        - Boot guestos
```
Comment (in1004kyu) [inline common/guest/command/command.h]: I will fix. boot (h) -> boot (b)

**Comment by kesl** [comment]
CI : Test PASSed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/128/

**Comment by kesl** [comment]
CI : Test FAILed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/129/

**Comment by in1004kyu** [comment]
retest this please

**Comment by kesl** [comment]
CI : Test FAILed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/130/

**Comment by in1004kyu** [comment]
I will re pull request.

**Pull request closed without merging**